frontend/src/lib/k8s/apiProxy.ts

```diff
@@ -1618,6 +1619,14 @@ export async function setCluster(clusterReq: ClusterRequest) {
 }
 
 export async function deleteCluster(cluster: string) {
```

Comment:
If we are removing the request to the backend then this function shouldn't be in the `apiProxy.ts` file.

Reply:
We are not removing request to the backend, it handles cases if it's a stateless cluster or not. If it's a stateless cluster it handles the indexDB part, if it's not, it makes request to the backend.